contrib/ci/check-null-false-returns.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python3
+# pylint: disable=invalid-name,missing-docstring,too-many-branches,too-many-statements
+#
+# Copyright (C) 2021 Richard Hughes <richard@hughsie.com>
+#
+# SPDX-License-Identifier: LGPL-2.1+
+
+import glob
+import sys
+
+
+def parse_rvif(line):
+
+    # remove whitespace
+    line = line.replace("\t", "")
+    line = line.replace(" ", "")
+
+    # find value
+    line = line.replace(",", "|")
+    line = line.replace(")", "|")
+    return line.rsplit("|", maxsplit=2)[1]
+
```

Comment:
I think you need to add something to strip comments no?  Because `/* foo bar * / g_return_val_if_fail(...);` is valid C isn't it?

Reply:
Sure, but we're doing a rsplit, so unless the comment is something like `g_return_val_if_fail(...); /* should use foo_cb()` I think it's fine.